.gitignore

```diff
@@ -3,3 +3,4 @@ out
 *.pyc
 .config
 .config.old
+*~
```

Comment:
What is the purpose of this addition?

Reply:
It ignores the previous-version files saved by some editors.  (joe, for instance, creates foo~ when foo is saved.)  It makes for cleaner output when you're creating a commit.

Reply:
That is not on topic I think. But I guess it's up to the owners of the repo.

Reply:
You should just make that a global ignore on your system!